app/views/decidim/vocdoni/admin/answers/new.html.erb

```diff
@@ -1,9 +1,12 @@
 <% add_decidim_page_title(t(".title")) %>
```

Comment:
Let' add a break-line (as it was before) so it easy to have the meta and body blocks visually separated. 

```suggestion
<% add_decidim_page_title(t(".title")) %>

```